src/app/components/textInput.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import { useState } from 'react';
+
+export const TextInput = () => {
```

Comment:
sounds like we may need a `TextInput.test.ts` ;)

Reply:
oi leave me alone